### PR TITLE
Changing clusters view to use similar classes as base console

### PR DESCRIPTION
**Pull request by crobby**
Now using the same css classes to make our page more closely
resemble the base console pages.

### CODE REVIEW

File: app/views/oshinko/clusters.html [hunk 1]
```diff
@@ -1,65 +1,76 @@
 <project-header class="top-header"></project-header>
-<project-page class="project-overview-page">
-<div class="container-cards-pf dashboard-cards">
-    <div class="card-pf card-pf-double" id="cluster-list">
-        <alerts alerts="alerts"></alerts>
-        <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
-        <div class="card-pf-heading">
+<project-page class="spark-clusters">
```
Comment (sub-mod) [inline app/views/oshinko/clusters.html]: where is the css for this class spark-clusters ?

Comment (crobby) [reply app/views/oshinko/clusters.html]: nowhere.  I should eliminate that.

File: app/views/oshinko/new-cluster.html [hunk 2]
```diff
@@ -1,33 +1,24 @@
...
+  </div>
 </div>
 <div class="modal-footer">
-    <button id="cancelbutton" class="btn btn-default btn-cancel1" translatable="yes" ng-click="cancelfn()">Cancel</button>
-    <button id="createbutton" class="btn btn-primary" translatable="yes" ng-click="complete(newCluster())">Deploy</button>
-</div>
-
+  <button id="cancelbutton" class="btn btn-default btn-cancel1" translatable="yes" ng-click="cancelfn()">Cancel</button>
+  <button id="createbutton" class="btn btn-primary" translatable="yes" ng-click="complete(newCluster())">Deploy</button>
+</div>
```
Comment (sub-mod) [inline app/views/oshinko/new-cluster.html]: newline required here

File: app/views/oshinko/scale-cluster.html [hunk 3]
```diff
@@ -2,14 +2,17 @@
...
+  <div class="form-error" ng-show="formError">{{ formError }}</div>
 </div>
 <div class="modal-footer">
-        <button id="cancelbutton" class="btn btn-default btn-cancel" translatable="yes" ng-click="cancelfn()">Cancel</button>
-        <button id="scalebutton" class="btn btn-primary" translatable="yes" ng-click="complete(scaleCluster(workerCount))">Scale</button>
-</div>
-
+  <button id="cancelbutton" class="btn btn-default btn-cancel" translatable="yes" ng-click="cancelfn()">Cancel</button>
+  <button id="scalebutton" class="btn btn-primary" translatable="yes" ng-click="complete(scaleCluster(workerCount))">Scale</button>
+</div>
```
Comment (sub-mod) [inline app/views/oshinko/scale-cluster.html]: newline required here

File: app/views/oshinko/new-cluster.html (hunk 2 above)
Comment (crobby) [reply app/views/oshinko/new-cluster.html]: done

File: app/views/oshinko/scale-cluster.html (hunk 3 above)
Comment (crobby) [reply app/views/oshinko/scale-cluster.html]: done

**Comment by sub-mod** [comment]
lgtm

**Pull request merged**